packages/bpk-component-loading-button/stories.js

```diff
@@ -293,78 +295,102 @@ storiesOf('bpk-component-loading-button', module)
   ))
   .add('Icon-only button', () => (
     <div>
-      <BpkLoadingButton iconOnly onClick={action('iconOnly button clicked')} />
+      <BpkLoadingButton iconOnly onClick={action('iconOnly button clicked')}>
+        <span className="visually-hidden">Search</span>
```

Comment:
Added these as `BpkLoadingButton` requires `children` prop in every instance so excluding it causes flow to report its invalid